pkg/bootstrapper/bootstrapper_test.go

```diff
@@ -176,7 +176,10 @@ func TestCreateKubeletConf(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			bs := winNodeBootstrapper{installDir: instDir}
+			bs := winNodeBootstrapper{
+				installDir:   instDir,
+				clusterDNSIP: "172.30.0.10",
```

Comment:
Where is this IP coming from? If this specific value is needed please make it a constant, if any value works please comment.

Reply:
The `172.30.0.10` value  is coming from the [template](https://github.com/openshift/windows-machine-config-bootstrapper/pull/306/files/39bc32c9bc83fd020254cfb3be0633f8f6b79980#diff-c28fa61c93e3f117dea1a808854d6b9309175dc85d833e2a67b6e33ff2d92bfaL174) used in the base case test.